package.json

```diff
@@ -3,31 +3,55 @@
   "version": "1.1.0",
   "description": "A Google Cloud Function for switching out quad words in print requests to ArcGIS Server",
   "repository": "https://github.com/agrc/serverless-print-proxy",
+  "license": "MIT",
+  "author": "Scott Davis <stdavis@utah.gov>",
   "main": "index.js",
+  "type": "module",
   "scripts": {
+    "format": "prettier . --write",
     "lint": "eslint *.js",
-    "test-debug": "node --inspect-brk ./node_modules/jest/bin/jest.js --runInBand",
-    "test-ci": "npm run lint && jest",
-    "test": "jest --watch",
-    "start": "node index.js"
+    "start": "NODE_ENV=development node index.js",
+    "start:prod": "NODE_ENV=production node index.js",
```

Comment:
**thought**: using [cross-env](https://github.com/kentcdodds/cross-env) here would allow this to run on windows.

Reply:
Do we care? Optimizing for windows development for JavaScript seems like something that's not worth our time. But maybe I'm missing something?

Reply:
I wouldn't call this optimizing for windows. I'd call it including windows. I think we should care a little bit because our development team is not entirely osx. Maybe a bridge we cross when it becomes an issue?